feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/staking/redeem/RedeemViewModel.kt

```diff
@@ -60,6 +60,10 @@ class RedeemViewModel(
         .flatMapLatest { interactor.assetFlow(it.stashAddress) }
         .share()
 
+    private val controllerAssetFlow = accountStakingFlow
+        .flatMapLatest { interactor.assetFlow(it.controllerAddress) }
+        .share()
+
```

Comment:
Why do we need separate asset flow for stash if we can just change `it.stashAddress` -> `it.controllerAddress` in `assetFlow` above?

Reply:
Yes, you are right. I though it will ruin the logic with 

`val amountLiveData = assetFlow.map { asset ->
        val redeemable = asset.redeemable

        redeemable.format() to asset.token.fiatAmount(redeemable)?.formatAsCurrency()
    }
        .inBackground()
        .asLiveData()`

but it works

Reply:
Done

Reply:
Yeah, the redeemable amount is the same in both controller and stash asset, so it is safe to use them interchangeably

Reply:
Yeah, the redeemable amount is the same in both controller and stash asset, so it is safe to use them interchangeably